x/common/types/rollapp_packet.go

```diff
@@ -38,6 +40,21 @@ func (r RollappPacket) ValidateBasic() error {
 }
 
 func (r RollappPacket) GetEvents() []sdk.Attribute {
+	var pd transfertypes.FungibleTokenPacketData
+	if len(r.Packet.Data) != 0 {
+		// It's okay if we can't get packet data
+		pd, _ = r.GetTransferPacketData()
+	}
+
+	var isAck bool
+	if len(r.Acknowledgement) != 0 {
+		ack, err := r.GetAck()
+		// It's okay if we can't get acknowledgement
+		if err == nil {
+			isAck = ack.Success()
+		}
+	}
```

Comment:
that's fine but I think `isAck` is confusing (cause it indicates if there was or wasn't an ack)
where we actually want to get if it was a successful or non successful ack (assuming one happened)

in the current case, if isAck is err, than `isAck` returns false (but there actually was an ack, jsut error ack)